### PR TITLE
Fix revdeps from Yojson PR

### PR DESCRIPTION
**Pull request by mseri**
See https://github.com/ocaml/opam-repository/pull/25994
Failures in commits

### CODE REVIEW

**Comment by Leonidas-from-XIV** [comment]
I think `sarif` and `safemonkey` need to have their `ppx_expect` dependency to be capped at `< v0.17` since [ppx_expect.common](https://github.com/janestreet/ppx_expect/tree/v0.16/common) existed in `v0.16` but is gone in `v0.17`.

File: packages/sarif/sarif.0.1.0/opam [hunk 1]
```diff
@@ -20,6 +20,7 @@ depends: [
   "timedesc"
   "ppx_jane"
   "ppx_deriving"
+  "ppx_expect"
```
Comment (mseri) [inline packages/sarif/sarif.0.1.0/opam:23]:
```suggestion
  "ppx_expect"  {< "v0.17"}
```

File: packages/sarif/sarif.0.2.0/opam [hunk 2]
```diff
@@ -20,6 +20,7 @@ depends: [
   "timedesc"
   "ppx_jane"
   "ppx_deriving"
+  "ppx_expect"
```
Comment (mseri) [inline packages/sarif/sarif.0.2.0/opam:23]:
```suggestion
  "ppx_expect"  {< "v0.17"}
```

**Comment by mseri** [comment]
Thanks!

**Pull request merged**